cmd/nerdctl/login.go

```diff
@@ -253,6 +253,12 @@ func tryLoginWithRegHost(ctx context.Context, rh docker.RegistryHost) error {
 		Host:   rh.Host,
 		Path:   rh.Path,
 	}
+	if rh.Path == "/v2" {
+		// If the path is using /v2 endpoint but lacks trailing slash add it
+		// https://docs.docker.com/registry/spec/api/#detail. Acts as a workaround
+		// for containerd issue https://github.com/containerd/containerd/blob/main/remotes/docker/config/hosts.go#L110
```

Comment:
Thanks, could you open the PR in the containerd repo

Reply:
Also, please sign the commit for DCO
https://github.com/apps/dco

(run `git commit -a -s --amend`, and make sure that the `Signed-off-by: NAME <EMAIL>` line with your real name is included in the commit message)

Reply:
I have raised https://github.com/containerd/containerd/pull/6470 with the needed DCO commit would you like me to close this PR?

Reply:
Yes, let me close this PR, assuming we can merge https://github.com/containerd/containerd/pull/6470

Reply:
Brilliant thanks will keep an eye out for any further changes in new PR